types/src/transaction/webauthn.rs

```diff
@@ -103,6 +84,21 @@ impl PartialAuthenticatorAssertionResponse {
         bcs::to_bytes(&self.signature).expect("Only unhandleable errors happen here.")
     }
 
+    /// This checks that the SHA3-256 of the [`signing_message(message)`](signing_message)
+    /// is equal to the actual `challenge` from `client_data_json` in `PartialAuthenticatorAssertionResponse`
+    /// and executes the checks defined in `verify_arbitrary_msg`
+    pub fn verify<T: Serialize + CryptoHash>(
```

Comment:
i believe this function is used in VM, so unless you can prove it doesn't change the result of old txns, i assume it will cause execution divergence...

Reply:
looks like you are mainly changing the order of some falliable steps...

Reply:
reverted changes to existing func.  RIP code reuse.